lib/elastic_apm/span.rb

```diff
@@ -25,6 +25,16 @@ class Span
     extend Forwardable
     include ChildDurations::Methods
 
+    # @api private
+    class Outcome
+      FAILURE = "failure"
+      SUCCESS = "success"
+
+      def self.from_status(code)
```

Comment:
```suggestion
      def self.from_http_status(code)
```

Although that might be obvious it doesn't hurt to be explicit